src/auth-service/models/AccessToken.js

```
@@ -222,6 +227,131 @@
+  },
+  async getExpiringTokens({ skip = 0, limit = 100, filter = {} } = {}, next) {
+    try {
+      const inclusionProjection = constants.TOKENS_INCLUSION_PROJECTION;
+      const exclusionProjection = constants.TOKENS_EXCLUSION_PROJECTION(
+        filter.category ? filter.category : "none"
+      );
+
+      if (!isEmpty(filter.category)) {
+        delete filter.category;
+      }
+      const currentDate = moment().tz(moment.tz.guess()).toDate(); // current date in the user's timezone
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)